CHANGELOG.md

```diff
@@ -2,28 +2,28 @@
 
 ## v0.10.1
 
-  * Fix a bug with `nil` decimals that happens on Ecto 3.11+.
```

Comment:
Can you please remove these whitespace changes? They're not necessary, and in addition I prefer the spaces there 🙃

Reply:
Done